assets/js/blocks/price-filter/editor.scss

```diff
@@ -0,0 +1,48 @@
+.components-disabled .wc-block-price-filter__range-input-wrapper .wc-block-price-filter__range-input {
+	&::-webkit-slider-thumb {
+		pointer-events: none !important;
+	}
+	&::-moz-range-thumb {
+		pointer-events: none !important;
+	}
+	&::-ms-thumb {
+		pointer-events: none !important;
+	}
+}
+.wc-block-price-slider {
+	.components-placeholder__instructions {
+		border-bottom: 1px solid #e0e2e6;
+		width: 100%;
+		padding-bottom: 1em;
+		margin-bottom: 2em;
+	}
+	.components-placeholder__label svg {
+		fill: currentColor;
+		margin-right: 1ch;
+	}
+	.components-placeholder__fieldset {
```

Comment:
Where are these styles used? I was trying to see if we could get rid of the `!important` rules but I can't find where they are being used.